test/it/user-get.js

```diff
@@ -43,5 +44,12 @@ describe('User API Tests', () => {
 
     // 4. Delete the user
     await utils.deleteUser(createdUser);
+
+    // 5. Verify user was deleted
+    try {
+      await client.getUser(createdUser.profile.login);
```

Comment:
client.getUser() throws an exception if not caught will fail. Not sure if I can assert in any other way, than to catch the exception and expect it to have a certain error message.